bluesky/callbacks/stream.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import time as ttime
+from collections import Iterable, ChainMap
+
+import streamz
+import jsonschema
+import numpy as np
+from event_model import DocumentNames, schemas
+
+from .core import CallbackBase
+from ..run_engine import Dispatcher
+from ..utils import new_uid
+
+logger = logging.getLogger(__name__)
+
+
+class LiveStream(CallbackBase):
+    """Create a secondary event stream of processed data"""
+    def __init__(self, in_node, out_node=None):
+        self.in_node = in_node
+        # Use the same node as an output if None is given
+        self.out_node = out_node or self.in_node
+        # The output should propagate the modified Event document down the
+        # pipeline
+        self.event = self.in_node.emit
+        self.out_node.sink(self.send_event)
+        # Public dispatcher for callbacks
+        self.dispatcher = Dispatcher()
+        # Local caches for internal use
+        self.seq_count = 0  # Maintain our own sequence count for this stream
+        self._stream_start_uid = None  # Generated start doc uid
+        self._raw_descriptors = dict()  # List of past raw event descriptors
+        self._described = list()  # List of raw event desc. we have translated
+        self._last_descriptor = None  # List of sent descriptors
+
+    def start(self, doc):
+        """Receive a stop document, re-emit it for this event stream"""
```

Comment:
docstring typo here

Reply:
Planning on another commit to fill out the docs. I'll make sure to fix this.